tools/CreateRuleFabricBot/CreateRuleFabricBot/Rules/PullRequestLabel/PullRequestLabelFolderCapability.cs

```diff
@@ -132,8 +139,41 @@ private List<CodeOwnerEntry> ReadOwnersFromFile()
             {
                 while ((line = sr.ReadLine()) != null)
                 {
-                    CodeOwnerEntry entry = CodeOwnerEntry.Parse(line);
-                    if (entry != null)
+                    // Does the line start with '# PRLabel: "label1", "label2"
+
+                    // Remove tabs and trim extra whitespace
+                    line = line.Replace('\t', ' ');
+                    line = line.Trim();
+
+                    // Empty line, move on
+                    if (string.IsNullOrEmpty(line))
+                    {
+                        continue;
+                    }
+
+                    CodeOwnerEntry entry = new CodeOwnerEntry();
+
+                    // if we have the moniker in the line, parse the labels
+                    if (line.IndexOf(CodeOwnerEntry.LabelMoniker, System.StringComparison.OrdinalIgnoreCase) >= 0)
+                    {
+                        entry.ParseLabels(line);
+
+                        // We need to read the next line
+                        line = sr.ReadLine();
```

Comment:
Are you making any assumptions about needing to convert tabs and trim the line where you might want to do that here as well?

Reply:
Good point! I'll see if we need to do this anywhere else.